rocAL/rocAL_pybind/examples/rocAL_api_video_pipeline.py

```diff
@@ -152,17 +160,11 @@ def main():
                 print("**************ends*******************")
                 print("**************", i, "*******************")
         data_loader.reset()
-    #Your statements here
+    # Your statements here
     stop = timeit.default_timer()
 
     print('\n Time: ', stop - start)
 
 
 if __name__ == '__main__':
```

Comment:
Add EOF in this file

Reply:
It shows an empty line at EOF. I'll add another line and check

Reply:
Its still not showing an EOF in file changes even with two lines at the end